test/CadlRanchProjects/type/property/nullable/src/Generated/Models/CollectionsByteProperty.Serialization.cs

```diff
@@ -32,12 +32,17 @@ internal static CollectionsByteProperty DeserializeCollectionsByteProperty(JsonE
                 }
                 if (property.NameEquals("nullableProperty"u8))
                 {
+                    if (property.Value.ValueKind == JsonValueKind.Null)
+                    {
+                        nullableProperty = null;
+                        continue;
+                    }
                     List<BinaryData> array = new List<BinaryData>();
                     foreach (var item in property.Value.EnumerateArray())
                     {
                         if (item.ValueKind == JsonValueKind.Null)
                         {
-                            array.Add(null);
+                            continue;
```

Comment:
Just continue? Looks not correct to me. I thought either throw error or just add null to array.

Reply:
we should never throw

Reply:
I will revert this